test/TestUtil.js

```diff
@@ -20,7 +20,8 @@ const { BuildInfo } = require('../lib/BuildInfo');
 
 exports.promiseLater = function (time, func) {
     if (func === undefined) {
-        func = () => {};
+        func = () => {
```

Comment:
Minor: I think the old version was better. If the linter is not happy about `{}`, maybe put a space in between `{ }`

Reply:
This are unfortunately auto changed by the IDE reverting them, I will check if there are others.